puppet/zulip_ops/files/prometheus/prometheus.yml

```diff
@@ -0,0 +1,34 @@
+# my global config
```

Comment:
Can you add a comment noting how this was generated and should be maintained?

Reply:
Cleaned up by removing the extraneous parts of it and adding more comments.  Let me know if it needs more still.